Formula/u-boot-tools.rb

```diff
@@ -1,8 +1,8 @@
 class UBootTools < Formula
   desc "Universal boot loader"
   homepage "https://www.denx.de/wiki/U-Boot/"
-  url "https://ftp.denx.de/pub/u-boot/u-boot-2019.10.tar.bz2"
-  sha256 "8d6d6070739522dd236cba7055b8736bfe92b4fac0ea18ad809829ca79667014"
+  url "http://ftp.denx.de/pub/u-boot/u-boot-2020.01.tar.bz2"
```

Comment:
Don't forget to change it back to `https` please 🙂

Reply:
oh my bad, thanks for spotting it out.